### DISCUSSION TITLE
Probably a dumb question but it's been working for the last year or more ....

### DISCUSSION THREAD

**Discussion by Acleacius** [Tyrrrz/YoutubeDownloader#562] [Q&A]
Even used it earlier today 12/13/24 but it stopped working, so downloaded the newer version v1.14.
Still won't start, think something happened and now win10  defender is blocking but it never says, no popups.
Tried running as Admin, still no luck.
Guess I need something to replace defender, maybe something like zone alarm? 🤷 

Thanks for any tips.

**Comment by Tyrrrz** [comment]
Does it report the reason it's blocking it from running?

**Comment by Acleacius** [comment]
Thanks for all your work and sharing! 🥂 

I really wish it did but it does not. It's as if it's not being allowed to run, hence me thinking it's something with defender.
I tried to look a little bit for a log file but no luck, and I'm just average at best working programs.
It's been working for at least a year, and stopped this week, oddly I think ms released a patch for win10.
Mine was set to no patching with 3rd party but it's like defender some how it finds away to turn itself back on or something.
In the past I've used zone alarm, and iirc it disabled defender and always gave a popup for something to run or be blocked.
It stopped working in the previous version and thought it might need to be updated, and installed the version from 3 days ago.
Still no luck. 😢
